composer.json

```diff
@@ -20,24 +20,21 @@
 	"minimum-stability": "stable",
 	"autoload": {
 		"psr-4": {
-			"Authorization\\": "src"
+			"Authorization\\": "src/"
 		}
 	},
 	"autoload-dev": {
 		"psr-4": {
-			"AuthTest\\": "tests/test_app/Plugin/AuthTest/src",
-			"Authorization\\Test\\": "tests",
-			"Authorization\\Test\\App\\": "tests/test_app/App",
-			"Cake\\Test\\": "./vendor/cakephp/cakephp/tests",
-			"TestApp\\": "tests/test_app",
-			"TestPlugin\\": "tests/test_app/Plugin/TestPlugin/src"
+			"Authorization\\Test\\": "tests/",
+			"Cake\\Test\\": "./vendor/cakephp/cakephp/tests/",
+			"TestApp\\": "tests/test_app/TestApp/",
+			"TestPlugin\\": "tests/test_app/Plugin/TestPlugin/src/"
 		}
 	},
 	"authors": [
-		{
-			"name": "Florian Krämer",
-			"homepage": "http://florian-kraemer.net",
-			"role": "Developer"
-		}
+			{
+					"name": "CakePHP Community",
+					"homepage": "https://github.com/cakephp/cakephp/graphs/contributors"
```

Comment:
Should it point to the plugin's contributors?

Reply:
Yes. This is copy paste fail.